src/models/addw4/repairs.md

```diff
@@ -0,0 +1,260 @@
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the battery:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Remove the five battery screws near the corners of the battery.
+
+![Battery screws](./img/battery.webp)
+
+3. Unplug the white connector connecting the battery to the motherboard.
+4. Remove and replace the battery.
+5. When plugging in the new battery, the red wire on the connector goes on the left, and the black wire goes on the right.
+
+## Replacing the CMOS battery:
+
+The CMOS battery supplies power to the system's CMOS chip. UEFI settings and the computer's hardware clock are stored on the CMOS. If your system doesn't boot, you can reset the CMOS to force a low-level hardware reset. If your clock is constantly resetting, it's likely your CMOS battery needs to be replaced.
+
+**Part numbers:**
+- The CMOS battery is a standard 3V 3Y3 CR2032W battery.
+
+**Tools required:** Cross-head (Phillips) screwdriver    
+**Time estimate:** 15 minutes    
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the CMOS battery:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover) and [remove the main battery](#replacing-the-battery).
+    - If you are not replacing the CMOS battery, then removing the main battery is optional.
+
+![CMOS battery](./img/cmos-battery.webp)
+
+2. Unplug the small white connector that connects the CMOS battery to the motherboard. If you are replacing the battery, gently pull from the sides to pry it up from where it's stuck to the case.
+3. To clear the CMOS, [disconnect the main battery](#replacing-the-battery) (if it's still installed), open the lid of the machine, and hold down the power button for at least 15 seconds to discharge any residual energy in the system.
+4. Reconnect the CMOS battery, reconnect the main battery, and replace the M.2 SSD.
+5. Replace the bottom panel and power up the Adder WS. The system may power itself off and on after initial boot; this is normal behavior when the CMOS has been reset.
+
+## Replacing the wireless card:
+
+Your Adder WS's WiFi and Bluetooth are both handled by the same module. It is a standard M.2 2230 slot with PCIe and USB interfaces (E-key).
+
+**Part number:**
+- The standard wireless card is an Intel `AX211NGW`.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>  
+
+### Steps to replace the WiFi/Bluetooth module:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover) and [remove the SSD from slot 1 (the innermost slot)](#replacing-an-m2nvme-ssd), if installed.
+    - You can optionally remove the [SSD from slot 2](#removing-the-bottom-cover) for easier access.
```

Comment:
This link should probably go to the SSD section, rather than the bottom cover section